source/api_reference/version-history/index.html.md.erb

```diff
@@ -37,7 +37,8 @@ Registers no longer have "indexes" ([RFC-20](https://github.com/openregister/reg
 These changes mean that:
 
 - you now access the resource at `/blobs/{blob-hash}` instead of `/items/{item-hash}`
-- the field `item-hash` is now named `blob-hash`
+- the field `item-hash` has been replaced by `blob-hash`
+- `blob-hash` is computed differently from the existing `item-hash` vaues, so if your code relies on [partial updates](/getting_updates), you should fetch all the data again after updating to this version of the API
```

Comment:
Looks fine except I'm not sure what's meant by "partial updates"

Reply:
I would probably just say "updates".